src/test/resources/dlsfls/internal_users_tlq.yml

```diff
@@ -0,0 +1,32 @@
+---
+_meta:
+  type: "internalusers"
+  config_version: 2
+  
+tlq_1337:
```

Comment:
Regarding all these internal users: why is it necessary to create a bunch of different internal users given that they all share the same `backend_roles`?

Reply:
Because they all have different access codes in the `user_access_codes ` index. The access codes in this index are used in the TLQ DLS query and determine which documents are visible for the user and which ones should be filtered.